frameworks/cassandra/universe/config.json

```diff
@@ -253,6 +253,58 @@
                   "minimum": 128000
                 }
               }
+            },
+            "rlimit_stack": {
+              "description": "Specifies RLIMIT_STACK, a value one greater than the maximum size of the initial thread's stack, in bytes.",
+              "type": "object",
+              "properties": {
+                "soft" : {
+                  "type": "integer",
+                  "description": "The  soft  limit  is  the  value that the kernel enforces for the corresponding resource.",
+                  "default": 102400,
+                  "minimum": 102400
+                },
+                "hard" : {
+                  "type": "integer",
+                  "description": "The  hard  limit  acts as a ceiling for the soft limit.",
+                  "default": 102400,
+                  "minimum": 102400
+                }
+              }
+            },
+            "rlimit_memlock": {
+              "description": "Specifies RLIMIT_MEMLOCK, the maximum number of bytes of memory that may be locked into RAM.",
+              "type": "object",
+              "properties": {
+                "soft" : {
+                  "type": "integer",
+                  "description": "The  soft  limit  is  the  value that the kernel enforces for the corresponding resource.",
```

Comment:
Please fix spacing and also include a note like `-1 represents unlimited amount of memory will be locked` or is it more along the lines of `highest amount of memory available`?